### PR TITLE
Add the release notes for 1.3.0 release

### PR DESCRIPTION
**Pull request by zelinh**
Signed-off-by: Zelin Hao <zelinhao@amazon.com>

### Description
Add the release notes for 1.3.0 release which combines the notes from core and plugins. 
 
### Issues Resolved
part of #889 
 
### Check List
- [x] Commits are signed per the DCO using --signoff 

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

**Comment by zelinh** [comment]
The PR for release notes of core is raised here: https://github.com/opensearch-project/OpenSearch/pull/2489.
I put the potential link in the certain section of this PR.

File: release-notes/opensearch-release-notes-1.3.0.md [hunk 1]
```diff
@@ -0,0 +1,632 @@
...
+
+OpenSearch and OpenSearch Dashboards 1.3.0 includes the following features, enhancements, bug fixes, infrastructure, documentation, maintenance, and refactoring updates.
+
+OpenSearch [Release Notes](https://github.com/opensearch-project/OpenSearch/blob/main/release-notes/opensearch.release-notes-1.3.0.md).
+
+OpenSearch Dashboards [Release Notes](https://github.com/opensearch-project/OpenSearch-Dashboards/blob/main/release-notes/opensearch-dashboards.release-notes-1.3.0.md).
+
+## FEATURES
+
+### Opensearch Anomaly Detection
```
Comment (kavilla) [inline release-notes/opensearch-release-notes-1.3.0.md:13]: Should be `OpenSearch`.

File: release-notes/opensearch-release-notes-1.3.0.md [hunk 2]
```diff
@@ -0,0 +1,632 @@
+# OpenSearch and Dashboards 1.3.0 Release Notes
```
Comment (bbarani) [inline release-notes/opensearch-release-notes-1.3.0.md:1]: Can we add release highlights for ml-commons?

File: release-notes/opensearch-release-notes-1.3.0.md (hunk 1 above)
Comment (bbarani) [reply release-notes/opensearch-release-notes-1.3.0.md:13]: Lets rename everywhere from opensearch -> OpenSearch

File: release-notes/opensearch-release-notes-1.3.0.md [hunk 3]
```diff
@@ -0,0 +1,632 @@
...
+* Bump version to 1.3 ([#106](https://github.com/opensearch-project/job-scheduler/pull/106))
+* Bumps cron-utils version ([111](https://github.com/opensearch-project/job-scheduler/pull/111))
+* Update copyright notices ([87](https://github.com/opensearch-project/job-scheduler/pull/87))
+* Add support for codeowners to repo ([100](https://github.com/opensearch-project/job-scheduler/pull/100))
+* Fixes copyright header ([127](https://github.com/opensearch-project/job-scheduler/pull/127))
+
+
+### Opensearch Ml Common
+* Bump version to 1.2 ([#90](https://github.com/opensearch-project/ml-commons/pull/90))
+* bump to 1.2.3 ([#110](https://github.com/opensearch-project/ml-commons/pull/110))
```
Comment (bbarani) [inline release-notes/opensearch-release-notes-1.3.0.md:561]: Why do we have 1.2.3 PR here?

File: release-notes/opensearch-release-notes-1.3.0.md (hunk 2 above)
Comment (zelinh) [reply release-notes/opensearch-release-notes-1.3.0.md:1]: I just added a brief line saying ml-commons now becomes part of the release. Any other recommendations about what to say here?

File: release-notes/opensearch-release-notes-1.3.0.md [hunk 4]
```diff
@@ -0,0 +1,637 @@
+# OpenSearch and Dashboards 1.3.0 Release Notes
+
+## Release Highlights
+
+### OpenSearch Ml Common
```
Comment (stockholmux) [inline release-notes/opensearch-release-notes-1.3.0.md:5]: ml common**s**

(with an "s") :)

File: release-notes/opensearch-release-notes-1.3.0.md [hunk 5]
```diff
@@ -0,0 +1,637 @@
...
+### OpenSearch Anomaly Detection Dashboards
+* Non blocker ([#202](https://github.com/opensearch-project/anomaly-detection-dashboards-plugin/pull/202))
+
+
+### OpenSearch Index Management
+* Continuous transforms ([#206](https://github.com/opensearch-project/index-management/pull/206))
+* Refactor IndexManagement to support custom actions ([#288](https://github.com/opensearch-project/index-management/pull/288))
+
+
+### OpenSearch Ml Common
```
Comment (stockholmux) [inline release-notes/opensearch-release-notes-1.3.0.md:31]: ml commons

File: release-notes/opensearch-release-notes-1.3.0.md [hunk 6]
```diff
@@ -0,0 +1,637 @@
...
+### OpenSearch Knn
+* Add Recall Tests ([#251](https://github.com/opensearch-project/k-NN/pull/251))
+* Change serialization for knn vector from single array object to collection of floats ([#253](https://github.com/opensearch-project/k-NN/pull/253))
+* Add ExtensiblePlugin to KNNPlugin ([#264](https://github.com/opensearch-project/k-NN/pull/264))
+* Add gradle task for running integ tests in remote cluster ([#266](https://github.com/opensearch-project/k-NN/pull/266))
+* Change benchmark ingest took metric to total time ([#268](https://github.com/opensearch-project/k-NN/pull/268))
+* Make doc and query count configurable in benchmark ([#270](https://github.com/opensearch-project/k-NN/pull/270))
+
+
+### OpenSearch Ml Common
```
Comment (stockholmux) [inline release-notes/opensearch-release-notes-1.3.0.md:117]: ml commons

File: release-notes/opensearch-release-notes-1.3.0.md [hunk 7]
```diff
@@ -0,0 +1,637 @@
...
+
+
+### OpenSearch Knn
+* Set default space type to L2 to support bwc ([#267](https://github.com/opensearch-project/k-NN/pull/267))
+* [BUG FIX] Add space type default and ef search parameter in warmup ([#276](https://github.com/opensearch-project/k-NN/pull/276))
+* [FLAKY TEST] Fix codec test causing CI to fail ([#277](https://github.com/opensearch-project/k-NN/pull/277))
+* [BUG FIX] Fix knn index shard to get bwc engine paths ([#309](https://github.com/opensearch-project/k-NN/pull/309))
+
+
+### OpenSearch Ml Common
```
Comment (stockholmux) [inline release-notes/opensearch-release-notes-1.3.0.md:243]: ml commons

File: release-notes/opensearch-release-notes-1.3.0.md [hunk 8]
```diff
@@ -0,0 +1,637 @@
...
+* Use Github App to trigger CI on backport PRs ([#288](https://github.com/opensearch-project/k-NN/pull/288))
+* Add auto delete workflow for backport branches ([#289](https://github.com/opensearch-project/k-NN/pull/289))
+* Updates Guava versions to address CVE ([#292](https://github.com/opensearch-project/k-NN/pull/292))
+* [CODE STYLE] Switch checkstyle to spotless ([#297](https://github.com/opensearch-project/k-NN/pull/297))
+* Switch main to 2.0.0-SNAPSHOT, update to Gradle 7.3.3 ([#301](https://github.com/opensearch-project/k-NN/pull/301))
+* Run CI on JDK 8 ([#302](https://github.com/opensearch-project/k-NN/pull/302))
+* Update numpy version to 1.22.1 ([#305](https://github.com/opensearch-project/k-NN/pull/305))
+
+
+### OpenSearch Ml Common
```
Comment (stockholmux) [inline release-notes/opensearch-release-notes-1.3.0.md:415]: ml commons

File: release-notes/opensearch-release-notes-1.3.0.md [hunk 9]
```diff
@@ -0,0 +1,637 @@
...
+
+### OpenSearch Index Management Dashboards Plugin
+* Add backport documentation link ([#161](https://github.com/opensearch-project/index-management-dashboards-plugin/pull/161))
+
+
+### OpenSearch Job Scheduler
+* Add release notes for 1.3.0.0 ([#143](https://github.com/opensearch-project/job-scheduler/pull/143))
+
+
+### OpenSearch Ml Common
```
Comment (stockholmux) [inline release-notes/opensearch-release-notes-1.3.0.md:483]: ml commons

File: release-notes/opensearch-release-notes-1.3.0.md [hunk 10]
```diff
@@ -0,0 +1,637 @@
...
+* Revert to windows 2019 for odbc CI ([#413](https://github.com/opensearch-project/sql/pull/413))
+
+
+## REFACTORING
+
+### OpenSearch Knn
+* Refactor benchmark dataset format and add big ann benchmark format ([#265](https://github.com/opensearch-project/k-NN/pull/265))
+
+
+### OpenSearch Ml Common
```
Comment (stockholmux) [inline release-notes/opensearch-release-notes-1.3.0.md:629]: ml commons

File: release-notes/opensearch-release-notes-1.3.0.md [hunk 11]
```diff
@@ -0,0 +1,637 @@
...
+* Feature latest observability ([#509](https://github.com/opensearch-project/observability/pull/509))
+* Live tail - Event analytics  ([#494](https://github.com/opensearch-project/observability/pull/494))
+* Add Events Flyout and Correlate Traces with logs ([#493](https://github.com/opensearch-project/observability/pull/493))
+* Merge Application Analytics into main ([#454](https://github.com/opensearch-project/observability/pull/454))
+
+
+### OpenSearch Performance Analyzer
+* Add .whitesource configuration file ([#119](https://github.com/opensearch-project/performance-analyzer/pull/119))
+* Add support for OPENSEARCH_JAVA_HOME ([#133](https://github.com/opensearch-project/performance-analyzer/pull/133))
+* Adding auto backport ([#146](https://github.com/opensearch-project/performance-analyzer/pull/146))
```
Comment (stockholmux) [inline release-notes/opensearch-release-notes-1.3.0.md:63]: Where should this go? Anything with workflows should not be considered new features but rather infrastructure.

(most of this section here, actually)

**Review by stockholmux — CHANGES_REQUESTED** [review]
- A few small changes regarding ml commons
- At a higher level, the categorization seems to be a little off. As an example, there is a lot of commits related to auto backport and they're spread throughout the different categories on different repos. Should this be all infrastructure?

**Review by gaiksaya — COMMENTED** [review]
Capitalize first letter of each line

File: release-notes/opensearch-release-notes-1.3.0.md [hunk 12]
```diff
@@ -0,0 +1,637 @@
+# OpenSearch and Dashboards 1.3.0 Release Notes
+
+## Release Highlights
+
+### OpenSearch Ml Commons
+* ml-commons now officially becomes part of the OpenSearch 1.3.0 release.
```
[the remark this answers is not in the document]
Comment (Naarcha-AWS) [reply release-notes/opensearch-release-notes-1.3.0.md:6]: We could say something like, "The new ML Commons plugin empowers users to train and  machine learning models as a part of the OpenSearch 1.3.0 release".

**Pull request merged**